### ISSUE TITLE
Article Alternative Layout resets to default when edited on fronted

### ISSUE THREAD

**Issue by jayblaq** [joomla/joomla-cms#7809]
#### Steps to reproduce the issue

Add an alternative layout file in the article overrides 

Go to the article and select the alternative layout from the options tab at the bottom

Next view the page and see the override is there.    

Next use the front end editor for the article and hit save, the article layout resets to default.

#### Expected result

The article layout should remain the same when editing on the front end.

**Comment by Trimud** [comment]
The main problem is that there is no option to choose alternative layout in frontend article editing.

Thanks to anyone willing to investigate and fix this issue.

**Comment by pepperstreet** [comment]
Good point! Does anyone have current infos about this issue. IMHO, a very crucial issue. 

Related questions raised in my mind:

- Does the problem occur with articles that got their alt-layout in the backend? 
- Does the problem occur with alt-MenuItems that have their own .XML files? 
- Does it need an alt-submission form as well?
- In general, what's best practice to set and keep alt-layout programmatically / in frontend? 

Any thoughts and experiences appreciated. 
Thanks in advance. 

PS: The override functionality will become even more important with the up-coming "custom-fields" feature! At least then, it has to work flawlessly.

**Comment by brianteeman** [comment]
Reset priority according to documented standards https://docs.joomla.org/Priority<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7809">issues.joomla.org/joomla-cms/7809</a>.</sub>

**Comment by conconnl** [comment]
@jayblaq i wasn't able to reproduce this issue.
Can you test it on the latest staging (3.6 beta 2)?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7809">issues.joomla.org/joomla-cms/7809</a>.</sub>

**Comment by brianteeman** [comment]
It has been a month since the request for confirmation without reply. I am closing this at this time. It can always be reopened if it is updated <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7809">issues.joomla.org/joomla-cms/7809</a>.</sub>

**Comment by amelcher** [comment]
Brian,

This is Adam (Nashville). I am experiencing this exact issue in 3.8.7.

**Steps to reproduce the issue**
Add an alternative layout file in the article overrides

Go to the article and select the alternative layout from the options tab in the backend

Refresh frontpage article with override to confirm alternative layout.

Next use the front end editor for the article and hit save, the article layout resets to default layout that is chosen in global configuration.

**Expected result**
The article layout should remain the same when editing on the front end. 

**Actual result**
There is no option to select alternative layout in front-end article editing. Once I confirmed that the front-end save change the layout - I then selected the alternative layout as the default option in the articles-global configuration. I then refreshed the front-end article page and saved it with the front-end editor.  The page did not change to Joomla default, rather it stayed the same as the alternative layout that I selected in global configuration.
